releasenotes/notes/bump-opa-rego-43-1-82ab33b58485e6b6.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    Bump ``github.com/open-policy-agent/opa`` to `v0.43.1 <https://github.com/open-policy-agent/opa/releases/tag/v0.43.1>`_ to patch CVE-2022-36085.
```

Comment:
The formatting of the version number and link look off to me, but I don't know what the correct markup is in this yaml.

Reply:
Here is what I used to check the formatting (this is some restructured text) https://livesphinx.herokuapp.com/

Reply:
But agreed, markdown would be way easier..